openedx/core/djangoapps/user_authn/views/tests/test_register.py

```diff
@@ -679,6 +679,8 @@ def assert_email_error(expected_error):
         self.assertLess(len(params["email"]), 254)
         self.assert_success(params)
 
+        self.client.logout()
```

Comment:
What's the logic behind logging out in the test here?

Reply:
newly registered users automatically login we would have to logout current user to do registration for the next user.

Reply:
These tests were written by edX and were falling before fixed these as we touched code in the same file.